conf/test.config

```
@@ -1,24 +1,25 @@
 /*
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Nextflow config file for running full-size tests
+    Nextflow config file for running representative-size tests
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-    Defines input files and everything required to run a full size pipeline test.
+    Defines input files and everything required to run a representative size pipeline test.
 
     Use as follows:
-        nextflow run nf-core/treeval -profile test_full,<docker/singularity> --outdir <OUTDIR>
+        nextflow run nf-core/treeval -profile test,singularity,sanger
```

Update at other places so that information represents the current pipeline state
```suggestion
        nextflow run sanger-tol/treeval -profile test,singularity -entry FULL
```

---

The test does run successfully. Thanks 👍🏼